cmd/podman/exists.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"github.com/containers/libpod/cmd/podman/libpodruntime"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	"os"
+)
+
+var (
+	existFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "q, quiet",
+			Usage: "silence output including error messages",
+		},
+	}
+)
+
+var (
+	imageExistsDescription = `
+	podman image exists 
+
+	Check if an image exists in local storage
+`
+
+	imageExistsCommand = cli.Command{
+		Name:         "exists",
+		Usage:        "Check if an image exists in local storage",
+		Description:  imageExistsDescription,
+		Action:       imageExistsCmd,
+		ArgsUsage:    "IMAGE-NAME",
+		OnUsageError: usageErrorHandler,
+	}
+)
+
+var (
+	containerExistsDescription = `
+	podman container exists 
+
+	Check if a container exists in local storage
+`
+
+	containerExistsCommand = cli.Command{
+		Name:         "exists",
+		Usage:        "Check if a containerexists in local storage",
```

Comment:
Missed a space between container and exists

Reply:
fixed